.gitignore

```diff
@@ -1,5 +1,6 @@
 **/vendor/
 composer.lock
+.scenarios.lock/
```

Comment:
If you want to use composer-test-scenarios to do highest/lowest testing (I recommend this, even for libraries), then you should commit composer.lock and .scenarios.lock. If you only want to do highest testing, then what you're doing here is fine.